CONTRIBUTING.md

```diff
@@ -0,0 +1,29 @@
+# Contribute to backend-swe-interview-questions
+
+## Welcome
+
+This repo contains technical questions (and answers) on several topics you might want to prepare for an interview for a Backend position. We believe in the power of open source and are always looking for contributors to help us improve the quality of the questions. Any contributions are welcome and appreciated.
+
+## What can I contribute?
+
+This repository contains a collection of technical questions and answers on several topics such as [Database](./answers/database.md), [Networking](./answers/networking.md), [Operating System](./answers/os.md),[Programming Paradigm](./answers/programming-paradigm.md), [Security](./answers/security.md), [Software Development Process](./answers/software-development-process.md) and many more to come. There are some types of contributions we are actively looking for:
+
+- **Answers**: There are many questions that have no answer yet, you can search the whole project `To be defined.`. If you have an answer to a question, please submit it as a pull request.
+
+<img width="578" alt="Screen Shot 2022-04-10 at 15 10 41" src="https://user-images.githubusercontent.com/8603085/162608993-1be7eb6f-7a45-4ebc-8c38-bf15ad9b05be.png">
+
+- **Questions**: If you have a question that you think should be included in the list, please add it to [README.md](./README.md). If you have an answer to that question, add it to [answers](./answers) folder as well. If you haven't had the answer yet, it's OK, just specify `To be defined.` to the corresponding answer file. We will seek for help for the answer from the community later.
```

Comment:
I think people might forget to add the `To be defined.` for new questions. Can you repharse it to make this point clearer?